scripts/dev.sh

```diff
@@ -53,15 +53,15 @@ function start_engine {
     --name $MESG_NAME \
     --tty \
     --label com.docker.stack.namespace=$MESG_NAME \
-    --label com.docker.stack.image=mesg/engine:local \
+    --label com.docker.stack.image=mesg/engine:dev \
     --env MESG_NAME=$MESG_NAME \
     --mount type=bind,source=/var/run/docker.sock,destination=/var/run/docker.sock \
     --mount type=bind,source=$MESG_PATH,destination=/root/.mesg \
     --network $MESG_NAME \
     --network name=$MESG_TENDERMINT_NETWORK \
     --publish $MESG_SERVER_PORT:50052 \
     --publish $MESG_TENDERMINT_PORT:26656 \
-    mesg/engine:local
+    mesg/engine:dev
```

Comment:
I would definitely prefer to build a local version that have a different tag than the one on docker hub (latest, dev and vX.X.X).
We actually use the dev version as a nighly or pre-prod. Maybe we should rename it to avoid confusion.